cobc/config.def

```diff
@@ -222,6 +222,9 @@ CB_CONFIG_SUPPORT (cb_stop_literal_statement, "stop-literal-statement",
 CB_CONFIG_SUPPORT (cb_stop_identifier_statement, "stop-identifier-statement",
 	_("STOP-identifier statement"))
 
+CB_CONFIG_SUPPORT (cb_stop_error, "stop-error",
```

Comment:
looks like it should have been "stop-error-statement" :-/

_runs away_

Reply:
Indeed. Fixed ;)